lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/ProcessJourneyEventHandler.java

```diff
@@ -396,6 +388,32 @@ private State executeStateTransition(
         return result.state();
     }
 
+    private State handleBackEvent(
+            JourneyState initialJourneyState, IpvSessionItem ipvSessionItem, String journeyEvent)
+            throws UnknownEventException, StateMachineNotFoundException, UnknownStateException {
+        if (BACK_EVENT.equals(journeyEvent) && !isBackEventDefinedOnState(initialJourneyState)) {
+            var previousJourneyState = ipvSessionItem.getPreviousState();
+
+            if (previousJourneyState != null
+                    && previousJourneyState
+                            .state()
+                            .equals("STRATEGIC_APP_TRIAGE/IDENTIFY_DEVICE")) {
```

Comment:
I'm a bit nervous about hardcoding this, what if we rename the state and forget to change it here?
Is there something else we could use to identify this state, perhaps a property on the state itself (eg. `skipBack`) (or this might point to the other idea of defining a bespoke response type for this)